sdk/src/main/java/com/deploygate/sdk/DeployGateCallback.java

```diff
@@ -6,16 +6,21 @@
  * {@link DeployGate#registerCallback(DeployGateCallback, boolean)} to listen.
  *
  * @author tnj
+ * @deprecated since 4.9.0. Use each extended callback interfaces instead.
  */
-public interface DeployGateCallback {
+@Deprecated
+public interface DeployGateCallback extends DeployGateInitializeCallback, DeployGateStatusChangeCallback, DeployGateUpdateAvailableCallback {
     /**
      * Callback to tell the finish of DeployGate service initialization
      * procedure.
      *
      * @param isServiceAvailable
      *         true if DeployGate service is available on the
      *         device.
+     * @deprecated since 4.9.0. Use {@link DeployGateInitializeCallback#onInitialized(boolean)} instead.
      */
+    @Deprecated
+    @Override
     public void onInitialized(boolean isServiceAvailable);
```

Comment:
We can also delete this method and delegate it to each extended class.
In this case, we cannot mark deprecated to methods in X.